### PR TITLE
Exec command

### PR DESCRIPTION
**Pull request by Jonahss**

### CODE REVIEW

File: lib/simctl.js [hunk 1]
```diff
@@ -6,8 +6,7 @@ import _ from 'lodash';
 
 const log = getLogger('simctl');
 
-async function simExec (command:string, timeout:number, args:Array = [],
-    env = {}) {
+async function simCommand (command:string, timeout:number, args:Array = [], env = {}, executingFunction) {
```
Comment (imurchie) [inline lib/simctl.js]: Does this actually work? Doesn't the linter complain about regular parameters coming after defaulted ones? Even if it doesn't, I don't think we should do it. `executingFunction = _.noop` would suffice.

Comment (Jonahss) [inline lib/simctl.js]: yeah works great.
I see what you mean though, I'll default it to simExec, how about that?

Comment (Jonahss) [inline lib/simctl.js]: (to get defaults like this, I think you can make a call like `simCommand(undefined, undefined, undefined, arg)`)

Comment (imurchie) [inline lib/simctl.js]: Yeah, defaulting to `simExec` sounds good.

**Comment by Jonahss** [comment]
@imurchie updated.

**Comment by imurchie** [comment]
Good to go!

**Pull request merged**